engine/src/main/java/org/hibernate/validator/internal/util/ReflectionHelper.java

```diff
@@ -259,36 +260,27 @@ public static Object getValue(Method method, Object object) {
 	}
 
 	/**
-	 * Indicates if the type is considered indexable (ie is an {@code Iterable}, an array or a {@code Map}).
+	 * Indicates if the type is considered a collection of elements (ie is an <code>Iterable</code>, an array or
+	 * a <code>Map</code>).
 	 *
-	 * @param type the type to inspect.
-	 *
-	 * @return Returns true if the type is indexable.
+	 * @param type the type to check.
+	 * @return returns <code>true</code> if <code>type</code> can be considered a collection of elements
```

Comment:
Do you think `@param` and `@return` add much value here?

I'm all for great documentation, but I think not stating the obvious also is a virtue as it reduces the time to read. Hence I often only put a method comment and omit the others. I'd only add them if there is something specific to say about them. That's what I'd put here:

    /**
     * Whether the given type represents a collection of elements or not (i.e. whether it's an
     * {@code Iterable}, {@code Map} or array type).
     */

I know it's nit-picking, but I thought I'd bring it up so we may converge (I've also changed my style over time, so you may see examples I wrote using the style I'm critizing here ;)).